sqlite/README.md

```diff
@@ -0,0 +1,354 @@
+# SQLite on Unikraft
+
+Build and run SQLite on Unikraft.
+Follow the instructions below to set up, configure, build and run SQLite.
+Make sure you installed the [requirements](../README.md#requirements).
+
+## Quick Setup (aka TLDR)
+
+For a quick setup, run the commands below.
+Note that you still need to install the [requirements](../README.md#requirements).
+Before everything, make sure you run the [top-level `setup.sh` script](../setup.sh).
+
+To build and run the application for `x86_64`, use the commands below:
+
+```console
+./setup.sh
+make distclean
+wget -O /tmp/defconfig https://raw.githubusercontent.com/unikraft/catalog-core/refs/heads/scripts/sqlite/scripts/defconfig/qemu.x86_64
+UK_DEFCONFIG=/tmp/defconfig make defconfig
+make -j $(nproc)
+test -f initrd.cpio || ./workdir/unikraft/support/scripts/mkcpio initrd.cpio ./rootfs/
+qemu-system-x86_64 \
+	-nographic \
+	-kernel workdir/build/sqlite_qemu-x86_64 \
+	-append "sqlite_qemu-x86_64 vfs.fstab=[ \"initrd0:/:extract::ramfs=1:\" ] -- /chinook.db 'SELECT * FROM Album LIMIT 10'" \
+	-initrd ./initrd.cpio
+```
+
+This will configure, build and run SQLite on Unikraft.
+You can see how to test it in the ["Test" section](#test).
+To close the virtual machine, see the instructions in the ["Close QEMU" section](#close-qemu).
+
+To do the same for `AArch64`, run the commands below:
+
+```console
+./setup.sh
+make distclean
+wget -O /tmp/defconfig https://raw.githubusercontent.com/unikraft/catalog-core/refs/heads/scripts/sqlite/scripts/defconfig/qemu.arm64
+UK_DEFCONFIG=/tmp/defconfig make defconfig
+make -j $(nproc)
+test -f initrd.cpio || ./workdir/unikraft/support/scripts/mkcpio initrd.cpio ./rootfs/
+qemu-system-aarch64 \
+	-nographic \
+	-machine virt \
+	-cpu max \
+	-kernel workdir/build/sqlite_qemu-arm64 \
+	-append "sqlite_qemu-arm64 vfs.fstab=[ \"initrd0:/:extract::ramfs=1:\" ] -- /chinook.db 'SELECT * FROM Album LIMIT 10'" \
+	-initrd ./initrd.cpio
+```
+
+Similar to the `x86_64` build, this will configure, build and run SQLite on Unikraft.
+
+> [!TIP]
+> To make the SQLite session interactive, as opposed to running a specific query, simply remove the `SELECT * FROM Album LIMIT 10` part from the `-append` argument. This will allow you to interact with the SQLite database directly using the shell. The argument would be:
```

Comment:
```suggestion
> To make the SQLite session interactive, as opposed to running a specific query, remove the `SELECT * FROM Album LIMIT 10` part from the `-append` argument.
> The only argument will the the `/chinook.db` path to the database file.
> This will allow you to interact with the SQLite database directly using the shell.
> The argument would be:
```